kratos/python/add_linear_solvers_to_python.cpp

```diff
@@ -237,6 +238,27 @@ void  AddLinearSolversToPython(pybind11::module& m)
     .def("__str__", PrintObject<DeflatedCGSolverType>)
     ;
 
+    using FallbackLinearSolverType = FallbackLinearSolver<SpaceType, LocalSpaceType>;
+    py::class_<FallbackLinearSolverType, FallbackLinearSolverType::Pointer, LinearSolverType>(m, "FallbackLinearSolver")
+    .def(py::init<Parameters>())
+    .def(py::init<LinearSolverType::Pointer, LinearSolverType::Pointer, Parameters>())
+    .def(py::init<const std::vector<LinearSolverType::Pointer>&, Parameters>())
+    .def("AddSolver", [](FallbackLinearSolverType& rSelf, LinearSolverType::Pointer pSolver) {
+        rSelf.AddSolver(pSolver);
+    })
+    .def("AddSolver", [](FallbackLinearSolverType& rSelf, LinearSolverType::Pointer pSolver, const Parameters ThisParameters) {
+        rSelf.AddSolver(pSolver, ThisParameters);
+    })
+    .def("GetSolvers", &FallbackLinearSolverType::GetSolvers)
+    .def("SetSolvers", &FallbackLinearSolverType::SetSolvers)
+    .def("GetResetSolverIndexEachTry", &FallbackLinearSolverType::GetResetSolverIndexEachTry)
```

Comment:
```suggestion
    .def("GetResetSolverEachTry", &FallbackLinearSolverType::GetResetSolverEachTry)
```

I would leave out the `Index`, to me this is confusing

Otherwise nice feature.

Do you need this a lot?

Reply:
We needed it, but we actually fixed the conditioning issue and iterative solver was Okay then 😗